bitnami/rabbitmq/Chart.yaml

```diff
@@ -23,4 +23,4 @@ name: rabbitmq
 sources:
   - https://github.com/bitnami/containers/tree/main/bitnami/rabbitmq
   - https://www.rabbitmq.com
-version: 11.3.1
+version: 11.3.3
```

Comment:
I think there was a little mistake here, bumping +2 patch versions. Should be `11.3.2`